files/etc/osmosis/99-one-click-osmosis

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+export TERM=xterm-256color
+echo "$(tput setaf 2) ██████╗ ███████╗███╗   ███╗ ██████╗ ███████╗██╗███████╗
+██╔═══██╗██╔════╝████╗ ████║██╔═══██╗██╔════╝██║██╔════╝
+██║   ██║███████╗██╔████╔██║██║   ██║███████╗██║███████╗
+██║   ██║╚════██║██║╚██╔╝██║██║   ██║╚════██║██║╚════██║
+╚██████╔╝███████║██║ ╚═╝ ██║╚██████╔╝███████║██║███████║
+ ╚═════╝ ╚══════╝╚═╝     ╚═╝ ╚═════╝ ╚══════╝╚═╝╚══════╝
+$(tput sgr0)
+To customize your droplet further, please run the following command:
+curl -sL https://get.osmosis.zone/install > i.py && python3 i.py
+
+For documentation visit: http://docs.osmosis.zone/
+For web app: https://app.osmosis.zone/
+
+To delete startup message: rm -rf $(readlink -f ${0})
+
```

Comment:
I would add a note here specifying which ports are open via `ufw` like the droplet docker image.

Reply:
No longer using ufw for now as per above discussion, if we decide to use ufw in the future we will for sure do this